### PR TITLE
KARAF-5397: Remove config for org.apache.karaf.shell.cfg from ssh fea…

### PR DESCRIPTION
**Pull request by graben**
…ture

### CODE REVIEW

**Comment by jbonofre** [comment]
R: @jbonofre

**Comment by graben** [comment]
Retest this please

**Comment by graben** [comment]
Something wrong with jenkins? Tried to build twice but without success.

**Comment by jbonofre** [comment]
Yes, there's a problem on Jenkins (not related to your PR):

```
java.io.IOException: failed to create temp directory at default location, most probably at: /tmp
```

I gonna deal to fix that.

File: assemblies/features/standard/src/main/feature/feature.xml [hunk 1]
```diff
@@ -663,12 +663,6 @@
     </feature>
 
     <feature name="ssh" description="Provide a SSHd server on Karaf" version="${project.version}">
-        <config name="org.apache.karaf.shell">
```
Comment (jbonofre) [inline assemblies/features/standard/src/main/feature/feature.xml]: This config is used by the default distribution, so I'm not sure it makes sense to remove this. I would update it instead.

Comment (graben) [reply assemblies/features/standard/src/main/feature/feature.xml]: Well, shell is also used by the default distribution and it also sets this config but actually gets overridden by ssh.

Comment (jbonofre) [reply assemblies/features/standard/src/main/feature/feature.xml]: Got it, let me test the PR.

**Pull request closed without merging**